packages/hardware-trezor/src/transformers/toTrezorTxIn.ts

```diff
@@ -0,0 +1,18 @@
+import * as trezor from 'trezor-connect';
+import { Cardano } from '@cardano-sdk/core';
+import { Transform } from '@cardano-sdk/util';
+import { TrezorTxTransformerContext } from '../types';
+import { resolvePaymentKeyPathForTxIn } from './key-paths';
+
+/**
+ * Transforms the given Cardano input transaction to the Trezor
+ * input transaction format using given the trezor input resolver.
+ */
+export const toTrezorTxIn: Transform<Cardano.TxIn, Promise<trezor.CardanoInput>, TrezorTxTransformerContext> = async (
+  txIn,
+  context?
+) => ({
+  path: (await resolvePaymentKeyPathForTxIn(txIn, context)) ?? undefined,
+  prev_hash: Buffer.from(txIn.txId).toString('hex'),
+  prev_index: txIn.index
+});
```

Comment:
Can we add `mapTxIns` here as well?
example: https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/hardware-ledger/src/transformers/txIn.ts#L35C14-L41

Reply:
yeah sure, i just wanted to push _something_ asap and implementing `mapTxIns` is just another transformer we can split up between multiple devs now :)

Reply:
👍